city_scrapers/settings/base.py

```diff
@@ -24,6 +24,9 @@
 # Disable cookies (enabled by default)
 COOKIES_ENABLED = False
 
+LOG_LEVEL='ERROR'
```

Comment:
We'll need this removed from the PR before merging. Fine for your local testing though